src/v2/Components/__tests__/SelectedCareerAchievements.jest.tsx

```diff
@@ -104,6 +104,21 @@ describe("SelectedCareerAchievements", () => {
 
     expect(text.length).toEqual(0)
   })
+
+  it("renders the Artists CV RouterLink regardless of career achievements", async () => {
+    // @ts-expect-error STRICT_NULL_CHECK
```

Comment:
These comments were added automatically by a script when we were migrating to strict type checking and needed a way to get the type-checker to pass without fixing every error that came up, so we should never add more, but rather address (and fix) the type error.

Reply:
@damassi  Do you have a suggestion on how to fix this typing? They are in two of the above unit tests as well, that you wrote from a year ago.

Reply:
I'm not sure, these `STRICT_NULL_CHECK` were added a month or so ago, automatically via a script, irrespective of any other tests or todos or tickets

Reply:
> Do you have a suggestion on how to fix this typing

What is the error?

Reply:
Argument of type '{ auctionResultsConnection: null; highlights: { partnersConnection: null; }; insights: null; slug?: string | undefined; id?: string | null | undefined; }' is not assignable to parameter of type '{ readonly highlights: { readonly partnersConnection: { readonly edges: readonly ({ readonly node: { readonly categories: readonly ({ readonly slug: string; readonly id: string | null; } | null)[] | null; readonly id: string | null; } | null; readonly id: string | null; } | null)[] | null; } | null; } | null; readon...'.
  Types of property 'slug' are incompatible.
    Type 'string | undefined' is not assignable to type 'string'.
      Type 'undefined' is not assignable to type 'string'.ts(2345)

Reply:
If this is something that is widespread, and is out of scope for this fix, I am going to add a JIRA ticket to this TODO so that we can track it and add in this change.

Reply:
In that case we can use `// @ts-ignore` with a FIXME pointing at the ticket 👍 (not sure what the issue is with the test, but its no big deal)

Reply:
The larger point is that anytime we see a _new_ 
```
// @ts-expect-error STRICT_NULL_CHECK
```
in code in a PR we can be sure that something has been copied from somewhere so we'll want to fix, or replace with `// @ts-ignore - FIXME: do something` 

In fact, any new `ts-expect-error` additions should fail on CI, but not sure why that isn't happening. Need to look into it.